.github/workflows/check-merge-conflict.yml

```
@@ -0,0 +1,16 @@
+name: Merge Conflict Check
+on:
+  push:
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check if prs are dirty
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        with:
+          dirtyLabel: has-conflicts
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          commentOnDirty: "This pull request has conflicts, please resolve those before we can evaluate the pull request."
```

Sounds a bit formal, no? :P Just a stupid suggestion.

```suggestion
          commentOnDirty: "This pull request has conflicts :pensive: Please resolve those before we can review the PR. Thanks!"
```

---

For ppl who are lazy like me: This is what `:pensive:` looks like: :pensive:

---

Let's keep this informative and neutral (no exclamations) and let's not use inconsistently cased abbreviations.

```suggestion
          commentOnDirty: |
              This pull request has conflicts.
              Please resolve those so we can review the pull request.
              Thanks.
```
I don't like the smiley in here. I think it's actually somewhat ridiculous. But if you must have it

```suggestion
          commentOnDirty: |
              This pull request has conflicts :pensive:
              Please resolve those so we can review the pull request.
              Thanks.
```

---

I don't have a strong opinion here :D 
Your call.

---

@JulianKniephoff, you have the remainder of the day to click on one suggestion – or to complain and suggest a new one – otherwise I'll pick one of those.

---

I actually can't click a button to accept the suggestions, so I'm just giving my vague opinion: I like emoji in online communication a lot, so I would tend to include one as well. I don't think :pensive: is the right one, though. I would probably just go for :frowning_face:. I can live with not having either of them as well, though.

---

I also do like emoji in online communication a lot. But I also get what you mean by "ridiculous". For example, I saw [this](https://medium.com/styled-components/announcing-styled-components-v5-beast-mode-389747abd987) release announcement recently and cringed a bit :P 

Just to add another data point: Rust bots (including the one that notifies of merge conflicts) use emojis, but not faces. For example :cloud_with_rain: or :umbrella: for merge conflicts, :evergreen_tree: for another thing. Dunno if that's any better.

---

Updated this and set a frowning face emoji